client/app/views/tutorial.coffee

```diff
@@ -25,7 +25,7 @@ module.exports = class Tutorial extends BaseView
     ]
 
     currentAppIndex: 0
-    appsToInstall: []
+    installedApps: []
```

Comment:
It's actually `installedAppsDuringTutorial`, but you are right it's misleading. Do you have any idea of better name (and shorter than `installedAppsDuringTutorial`)?